flask_potion/__init__.py

```diff
@@ -182,8 +193,7 @@ def add_route(self, route, resource, endpoint=None, decorator=None):
 
         view = self.output(view_func)
```

Comment:
Why not simply move these three lines into `_register_view()`? Avoids code duplication.